solutions/community-deploy/cloud/AWS/user.tfvars

```diff
@@ -18,3 +18,4 @@
 os_version          = "<OS VERSION>"
 region              = "<AWS REGION>"
 security_group_cidr = "<YOUR PUBLIC IP CIDR>"
```

Comment:
Please add EBS count also

Reply:
Its already changed as per https://github.com/Seagate/cortx-re/pull/1152/files#diff-206aed9711dcaa610102845194d4cb6f3d0b6a24a731b10715c79119e677dbe0

Reply:
Yes but this PR seems to be removing it.  Refer - https://github.com/Seagate/cortx-re/blob/mukul-multinode-2/solutions/community-deploy/cloud/AWS/user.tfvars

Reply:
https://github.com/Seagate/cortx-re/blob/mukul-multinode-2/solutions/community-deploy/cloud/AWS/user.tfvars  - Not showing EBS volume details

Reply:
fixed